CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 Change Log
 ==========
 
+## Version 1.0.0.alpha.1
+
+_2020-07-07_
+
+* Breaking: `Worker`s have now moved from the core `Workflow` library to `WorkflowReactiveSwift`. (#12)
+* Breaking: `Worker`s are now keyed based on type. ([workflow/1021](https://github.com/square/workflow/issues/1021))
+* `Swift` and [`Kotlin`](https://github.com/square/workflow-kotlin) libraries now live in seperate repos.
+* `Worker`s are now built on a more fundamental concept called `SideEffect`.  ([workflow/1021](https://github.com/square/workflow/issues/1021))
+* Introduce `WorkflowRxSwift` to build a `RxSwift` `Worker`. (#27)
+* Several APIs have been updated to be consistent. (#9)
+  * `MyWorkflow().rendered(with: context)` to `MyWorkflow().rendered(in: context)`
+  * `context.awaitResult(for: MyWorker())` to `MyWorker().running(in: context)`
```

Comment:
We should mention that the old versions are still around but are deprecated and will be removed for the beta.